chore(bundle): avoid imports in "global" format

pkozlowski-opensource opened the pull request:
If an "empty" file (like angular2/template.js) is imported
it is auto-detected as the one using "global" format.
This is incorrect as the entire angular2 build output is in
the ES6 format. The fix consists of forcing the es6 format
for files where auto-detection fails.

Also see: https://github.com/systemjs/systemjs/wiki/Module-Format-Support

pkozlowski-opensource commented:
@rkirov this currently breaks loading of bundles created from `master` with a minimal System.import loader.

rkirov commented:
There are few other empty files in the same dir - `view.js` and `pipes.js`
@petebacondarwin what is the purpose of having those empty files and what would be a good way to find all of them?

pkozlowski-opensource commented:
@rkirov yeh, saw other empty files, bit this was the only one referenced from https://github.com/angular/angular/blob/master/modules/angular2/angular2.js

rkirov commented:
Maybe we should just unreference it from the public exports. I dont' see a purpose or reexporting an empty file that is meant to be kept empty (if I understand its purpose as a djeni placeholder)

pkozlowski-opensource commented:
Yeh, this was my another option :-) @petebacondarwin ?

petebacondarwin commented:
My understanding is that these files are public API modules for angular. I don't think they should be empty. Instead they should contain re-exports of things in private inner modules.

rkirov commented:
ok, so the plan is to move the API exports into those files, it just too early now. Pawel, I saw remove reexport in `angular2/angular2`. When the API gets moved that file the reexport will get added and we would have no problems.

pkozlowski-opensource commented:
@rkirov right, works for me. I'm going to update the PR to remove the import

pkozlowski-opensource committed:
chore(bundle): avoid imports in "global" format

If an "empty" file (like angular2/template.js) is imported
it is auto-detected as the one using "global" format by the
system builder. This is incorrect as the entire angular2 build
output is in the ES6 format.

Removing empty import till it has some content.

pkozlowski-opensource commented:
@rkirov updated
